src/data/complex/exponential.lean

```diff
@@ -862,6 +862,13 @@ of_real_inj.1 $ by simpa using cos_square x
 lemma sin_square : sin x ^ 2 = 1 - cos x ^ 2 :=
 eq_sub_iff_add_eq.2 $ sin_sq_add_cos_sq _
 
+/-- The real definition of `sinh`-/
```

Comment:
```suggestion
/-- The definition of `sinh` in terms of `exp` -/
```

Reply:
I feel like the comment should refer to the fact it's `real`, but I'm not sure how to phrase the doc comment for this or `cosh`. The complex one is like this:

```
The complex hyperbolic sine function, defined via `exp`
```

Maybe something similar here?